src/store/actionCreators/actions.js

```diff
@@ -0,0 +1,20 @@
+import {
+  ADD_TODO,
+  DELETE_TODO,
+  TOGGLE_TODO,
+} from "../actionsTypes/actionTypes";
+
+export const addTodo = (message) => ({
+  type: ADD_TODO,
+  payload: { message },
+});
+
+export const toggleTodo = (id, isCompleted) => ({
+  type: TOGGLE_TODO,
+  payload: { id, isCompleted },
+});
+
+export const deleteTodo = (id) => ({
+  type: DELETE_TODO,
+  payload: { id },
+});
```

Comment:
Create simpleActionGenerator func that will return a new such object {payload: payloadFromParam, type: typeFromParam}

Reply:
Will be a generic action creator:
const actionCreator1 = (payload) => simpleActionGenerator(TYPE, payload)